index.d.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Add search paths for looking up symbol files.
+ */
+export function addSymbolPath(...paths: string[]): void
+
+/**
+ * Get the stack trace from `minidumpFilePath`
+ * The `callback` would be called with `callback(error, report)` upon completion.
+ */
+export function walkStack(
+  minidump: string,
+  callback: (err: Error, result: string) => void,
+  commandArgs?: string[]
+): void
+export function walkStack(
+  minidump: string,
+  symbolPaths: string[],
+  callback: (err: Error, result: string) => void,
+  commandArgs?: string[]
+): void
+
```

Comment:
Can you add a type signature for `moduleList`?

```ts
type ModuleInfo = {
  version: string
  name: string
  pdb_file_name?: string
  debug_identifier?: string
}
export function moduleList(
  minidump: string,
  callback: (err: Error, result: ModuleInfo[]) => void
): void
```

Reply:
Added